L24-c++-extensible-api.md

```diff
@@ -0,0 +1,45 @@
+Title
+----
+* Author(s): makdharma
+* Approver: vpai
+* Status: Draft
+* Implemented in: https://github.com/grpc/grpc/pull/14517
+* Last updated: Feb 26, 2018
+* Discussion at: <google group thread> (filled after thread exists)
+
+## Abstract
+
+This proposal will enable extending functionality of GRPC C++ API.
+
+## Background
+
+All GRPC C++ API classes are currently marked "final". The rationale is that
```

Comment:
Please use backquotes rather than doublequotes to indicate language features since that allows them to be typeset to look like code.

Reply:
done